server/server.js

```diff
@@ -0,0 +1,44 @@
+require('dotenv').config()
+
+const express          = require('express'),
+      port             = process.env.PORT,
+      path             = require('path'),
+      webpack          = require('webpack'),
+      config           = require('../webpack.config'),
+      compiler         = webpack(config),
+      cors             = require('cors')
+      app              = express(),
+      spotifyConnector = require('./spotify-connector'),
+      request          = require('request')
+      saveAcessToken   = require('./spotify-connector'),
```

Comment:
Hmm tutaj chyba coś jest  nie tak bo importujesz z tego pliku dwa razy, jako 2 różne rzeczy. Imho też lepiej poukładać importy 3rd party -> własne pliki.

Reply:
to pierwsze to smiec yup, wywalilem. I przenioslem importy